osu.Game/Screens/Edit/Editor.cs

```diff
@@ -329,6 +331,10 @@ private void load(OsuConfigManager config)
                                             new ToggleMenuItem(EditorStrings.ShowHitMarkers)
                                             {
                                                 State = { BindTarget = editorHitMarkers },
+                                            },
+                                            new ToggleMenuItem(EditorStrings.SeekToHitObject)
```

Comment:
This is in the "view" menu, but it isn't really a "view" option, so it kind of doesn't belong. Not sure what to do about this. @ppy/team-client your thoughts would be appreciated.